src/wp-admin/comment.php

```diff
@@ -41,7 +41,7 @@
 	$comment    = get_comment( $comment_id );
 
 	// Prevent actions on a comment associated with a trashed post.
-	if ( $comment && 'trash' === get_post_status( $comment->comment_post_ID ) ) {
+	if ( $comment instanceof WP_Comment && 'trash' === get_post_status( $comment->comment_post_ID ) ) {
```

Comment:
Not a criticism, but a question for my personal edification: Is there a reason that `is_a( $comment, 'WP_Comment' )` isn't being used?

Reply:
`instanceof` is typically more performant. There's _many_ debates about whether or not to include micro-optimizations, which ultimately result in personal preference, usually in terms readability, and in that sense, I prefer `instanceof` as it's _closer_ to being grammatically correct.

In the context of WordPress, given its scale, I try to implement any optimization that comes to mind when I'm writing a patch.

Reply:
Interesting.  Thanks for the response.  I learned something today!